src/NzbDrone.Core/MetadataSource/BookInfo/BookInfoProxy.cs

```diff
@@ -878,7 +878,8 @@ private static Book MapBook(WorkResource resource)
 
             if (resource.Books != null)
             {
-                book.Editions = resource.Books.Select(x => MapEdition(x)).ToList();
+                // "ForeignWorkId != null" to prevent retriving unref Goodreads' books (alias editions), and invoquing duplicates (gh #2153, #2082, #1573, #1514)
```

Comment:
Something along the lings of `Only fetch editions where ForeignWorkId is available` reads better and eliminates the spelling mistakes and unnecessary github issue references.

Reply:
Hi, it's just a modification for the comment line ? Sorry, English is not my natural language... so, of course, feel free to change anything you think could be better done... I just tried to contribute to solve this issue (I'm not attached to the way it will be really done, and neither the comment I put, just to try to explain why I made this)
Do I have to do something on my side to see this implemented ? 
Thanks

Reply:
You do not need to tag any devs. They are aware of this pull request, and will pull it when and if they deem it appropriate.